DCC.schema.json

```diff
@@ -3,32 +3,53 @@
   "$id": "https://id.uvci.eu/DCC.schema.json",
   "title": "EU DCC",
   "description": "EU Digital Covid Certificate",
-  "$comment": "Schema version 1.2.1",
-  "required": [
-    "ver",
-    "nam",
-    "dob"
-  ],
+  "$comment": "Schema version 1.3.0",
   "type": "object",
+  "oneOf": [
+    {
+      "required": [
+        "ver",
+        "nam",
+        "dob",
+        "v"
+      ]
+    },
+    {
+      "required": [
+        "ver",
+        "nam",
+        "dob",
+        "t"
+      ]
+    },
+    {
+      "required": [
+        "ver",
+        "nam",
+        "dob",
+        "r"
+      ]
+    }
+  ],
   "properties": {
     "ver": {
       "title": "Schema version",
       "description": "Version of the schema, according to Semantic versioning (ISO, https://semver.org/ version 2.0.0 or newer)",
       "type": "string",
       "pattern": "^\\d+.\\d+.\\d+$",
       "examples": [
-        "1.2.1"
+        "1.3.0"
       ]
     },
     "nam": {
-      "description": "Surname(s), given name(s) - in that order",
+      "description": "Surname(s), forename(s) - in that order",
       "$ref": "https://id.uvci.eu/DCC.Core.Types.schema.json#/$defs/person_name"
     },
     "dob": {
       "title": "Date of birth",
       "description": "Date of Birth of the person addressed in the DCC. ISO 8601 date format restricted to range 1900-2099",
       "type": "string",
-      "pattern": "^(19|20)\\d\\d(-\\d\\d){0,2}$",
+      "pattern": "^((19|20)\\d\\d(-\\d\\d){0,2}){0,1}$",
```

Comment:
Please explain this chain for me and why the old pattern was not correct.

Reply:
Is one formulation of a regex for allowing a completely empty date of birth. Works and was kindly contributed by @miguelzapaton. Of course, there are other ways of formulating the same regex, but this one works so I'm fine with it as it was originally contributed.
Spec:
> If the date of birth is not known even partially, the field MUST be set to an empty string "". T